integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java

```diff
@@ -89,211 +106,110 @@ public void cleanupOperatorAndDomain() throws Exception {
-    }.getClass().getEnclosingMethod().getName();
-    logTestBegin(testMethod);
-    OP_NS = "weblogic-operator221";
-    OP_DEP_NAME = "operator-upgrade221";
-    OP_SA = "operator-sa221";
-    DOM_NS = "weblogic-domain221";
-    DUID = "operatordomain221";
-    setupOperatorAndDomain("release/2.2.1", "2.2.1");
-    upgradeOperator(false);
-    testCompletedSuccessfully = true;
-    LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethod);
-  }
-
-  /**
-   * Test for upgrading Operator from release 2.3.0 to develop branch.
-   *
-   * @throws Exception when upgrade fails
-   */
-  @Test
-  public void testOperatorUpgradeFrom2_3_0() throws Exception {
+  public void testOperatorUpgradeFrom2_5_0() throws Exception {
     Assumptions.assumeTrue(QUICKTEST);
     testCompletedSuccessfully = false;
     String testMethod = new Object() {
     }.getClass().getEnclosingMethod().getName();
     logTestBegin(testMethod);
-    OP_NS = "weblogic-operator230";
-    OP_DEP_NAME = "operator-upgrade230";
-    OP_SA = "operator-sa230";
-    DOM_NS = "weblogic-domain230";
-    DUID = "operatordomain230";
-    setupOperatorAndDomain("release/2.3.0", "2.3.0");
-    upgradeOperator(false);
+    OP_NS = "weblogic-operator250";
+    DOM_NS = "weblogic-domain250";
+    namespaceList.append(OP_NS);
+    namespaceList.append(" ").append(DOM_NS);
+    OP_DEP_NAME = "operator-upgrade250";
+    OP_SA = "operator-sa250";
+    DUID = "operatordomain250";
+    setupOperatorAndDomain("release/2.5.0", "2.5.0");
+
+    // Save the CreateTimeStamp for the server pod(s) to compare with 
+    // CreateTimeStamp after upgrade to make sure the pod(s) are not re-stated
+    managed1CreateTimeStamp = TestUtils.getCreationTimeStamp(DOM_NS,DUID + "-managed-server1");
+    managed2CreateTimeStamp = TestUtils.getCreationTimeStamp(DOM_NS,DUID + "-managed-server2");
+    adminCreateTimeStamp = TestUtils.getCreationTimeStamp(DOM_NS,DUID + "-admin-server");
+    upgradeOperator();
     testCompletedSuccessfully = true;
     LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethod);
   }
 
   /**
    * Upgrades operator to develop branch by using the helm upgrade.
    *
-   * @param restart boolean parameter used to determine if a restart of domain is checked
    * @throws Exception when upgrade fails or basic usecase testing or scaling fails.
    */
-  private void upgradeOperator(boolean restart) throws Exception {
+  private void upgradeOperator() throws Exception {
     operator.callHelmUpgrade("image=" + OP_TARGET_RELEASE);
-    if (restart) {
-      checkDomainRollingRestarted();
-    }
-    checkOperatorVersion();
+    checkCrdVersion();
+    checkDomainNotRestarted();
     testClusterScaling(operator, domain, false);
   }
 
   /**
-   * checks the expected version of the upgraded operator in a loop. In Jenkins it takes nearly 8
-   * minutes to show the updated value of the domain CRD.
+   * Checks the expected Upgraded Version of CustomResourceDefintion (CRD) 
```

Comment:
CRD's are not namespaced resoruces, so this doc is wrong.

Reply:
Remove the line containing default ns